src/infohud.asm

```diff
@@ -1471,5 +1482,5 @@ HexToNumberGFX2:
     dw #$0C09, #$0C00, #$0C01, #$0C02, #$0C03, #$0C04, #$0C05, #$0C06, #$0C07, #$0C08
 
 print pc, " infohud bank80 end"
-warnpc $80FFC0 ; header
+warnpc $80FFB0 ; header
```

Comment:
SM starts its game header at $80FFC0, with the game name. I skips the first half of the registration data and fills it with $FF.

Reply:
SM game header starts at FFC0, but there is the extended header starting at FFB0, figure we should stay out of that region.